package.json

```diff
@@ -42,6 +43,7 @@
     "license.txt",
     "package.json",
     "README.md",
+    "dist/index.d.ts",
```

Comment:
this will make sure that the types will be included in the package once a new npm release is published